src/components/rockets/Rockets.jsx

```diff
@@ -1,5 +1,17 @@
-const Rockets = () => (
-  <div>Rockets go here</div>
-);
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { getRockets } from '../../redux/rockets/rockets';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getRockets);
+  }, []);
```

Comment:
Fetching the data from the API shouldn't happen inside the component, because every time the component re-renders (using nav links for example) the `useEffect` will get triggered every time and you will lose the changes you made to the state because it will be loaded again from scratch from the API.
check [the issue's note](https://github.com/ha-manel/Space-Travelers-Hub/issues/17) for reference

[Solution]
You can fetch the data from the API in `app.js` instead, this way it will load data from the API only once when it's rendered for the first time.